tests/e2e/pageObjects/monitor-page.ts

```diff
@@ -0,0 +1,68 @@
+    //DECLARATION OF TYPES: DOM ELEMENTS and UI COMPONENTS
+    //*Assign the 'Selector' type to any element/component nested within the constructor.
+    //------------------------------------------------------------------------------------------
+
+    expandMonitor: Selector
+    monitorArea: Selector
+    runMonitorToggle: Selector
+    startMonitorButton: Selector
+    clearMonitorButton: Selector
+    monitorIsStoppedText: Selector
+    monitorIsStartedText: Selector
+    hideMonitor: Selector
+    closeMonitor: Selector
+    monitorWarningMessage: Selector
+    monitorCommandLinePart: Selector
+
+    constructor() {
+        //-------------------------------------------------------------------------------------------
+        //DECLARATION OF SELECTORS
+        //*Declare all elements/components of the relevant page.
+        //*Target any element/component via data-id, if possible!
+        //*The following categories are ordered alphabetically (Alerts, Buttons, Checkboxes, etc.).
+        //-------------------------------------------------------------------------------------------
+        //BUTTONS
+        this.expandMonitor = Selector('[data-testid=expand-monitor]');
+        this.monitorArea = Selector('[data-testid=monitor]');
+        this.runMonitorToggle = Selector('[data-testid=toggle-run-monitor]');
+        this.startMonitorButton = Selector('[data-testid=start-monitor]');
+        this.clearMonitorButton = Selector('[data-testid=clear-monitor]');
+        this.monitorIsStoppedText = Selector('[data-testid=monitor-stopped]');
+        this.monitorIsStartedText = Selector('[data-testid=monitor-started]');
+        this.hideMonitor = Selector('[data-testid=hide-monitor]');
+        this.closeMonitor = Selector('[data-testid=close-monitor]');
+        this.monitorWarningMessage = Selector('[data-testid=monitor-warning-message]');
+        this.monitorCommandLinePart = Selector('[data-testid=monitor] span');
+    }
+    /**
+     * Check specific command in Monitor
+     * @param command A command which should be displayed in monitor
+     * @param parameters An arguments which should be displayed in monitor
+     */
+    async checkCommandInMonitorResults(command: string, parameters?: string[]): Promise<void> {
+        const commandArray = command.split(' ');
+        for (const value of commandArray) {
+            await t.expect(this.monitorCommandLinePart.withText(value).exists).ok('');
+        }
+        if (!!parameters) {
+            for (const argument of parameters) {
+                await t.expect(this.monitorCommandLinePart.withText(argument).exists).ok('');
+            }
+        }
+    }
+    /**
+     * Start monitor function
+     */
+    async startMonitor(): Promise<void> {
+        await t.click(this.expandMonitor);
+        await t.click(this.startMonitorButton);
+        //Check for "info" command that is sent every 5 seconds
+        await t.wait(5000);
```

Comment:
Can you explain?

Reply:
BE automatically sends "info" command every 5 seconds. Starting monitor I am waiting 5 seconds just to get this command logged inside of monitor (just to make sure that monitor is started). 
I had "checkCommandInMonitorResults" function that checked if a specified command appeared in monitor (without timeouts)  but for the one "info" command used "t.wait". 
Fixed according to your comment (now { timeout: 5000 } is set for any command in the function "checkCommandInMonitorResults").